server/echo_handler.cc

```diff
@@ -0,0 +1,19 @@
+#include "echo_handler.h"
+#include "httpRequest.h"
+#include "httpResponse.h"
+#include "http.h"
+#include <string>
+
+EchoHandler::Status EchoHandler::Init(const std::string& uri_prefix,
+				      const NginxConfig& config) {
+  return OK;
+}
+
+RequestHandler::Status EchoHandler::HandleRequest(const Request& request,
```

Comment:
Consistent with the above, or consistent in terms of RequestHandler and EchoHandler (all on line 12)? If it's the latter case, I had originally given EchoHandler identical enums, but then thought it might be smoother to just use the parent class's.